### PR TITLE
Blocumen added in teams section byline

### PR DESCRIPTION
**Pull request by nikhil3000**

### CODE REVIEW

File: server/views/components/team.html [hunk 1]
```diff
@@ -1,436 +1,696 @@
 <style>
-
```
Comment (anistark) [inline server/views/components/team.html]: Can you not change these formatting?

**Comment by nikhil3000** [comment]
Hey, Actually I did not change any formatting, it's just that my auto-indent of vs code moved everything to new lines. But all the code in intact.

**Comment by anistark** [comment]
> Hey, Actually I did not change any formatting, it's just that my auto-indent of vs code moved everything to new lines. But all the code in intact.

Yeah. Which makes it impossible to review the code. 
Can you revert to the original format? And change only the text that's needed to be? 

I'll put up a contributing guideline later to avoid this confusion.

**Comment by anistark** [comment]
Done in [a separate commit](https://github.com/blockchainedindia/website/commit/2dc98406a390b64e7f28209a3edfc6cb65f6805a).

**Pull request closed without merging**